src/App.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import "./App.css";
+async function searchNews(q) {
+  q = encodeURIComponent(q);
+  const response = await fetch(
+    `https://bing-news-search1.p.rapidapi.com/news/search?freshness=Day&textFormat=Raw&safeSearch=Strict&q=${q}`,
+    {
+      method: "GET",
+      headers: {
+        "x-rapidapi-host": "bing-news-search1.p.rapidapi.com",
+        "x-rapidapi-key": "9359f38d68msh7ec75863b6abbddp1cd680jsn6059448d81f9",
+        "x-bingapis-sdk": "true",
+      },
+    }
+  );
+  const body = await response.json();
+  return body.value;
+}
+function App() {
+  const [query, setQuery] = React.useState("Input");
```

Comment:
try using the new way to write usestae:  https://beta.reactjs.org/apis/usestate#adding-state-to-a-component